demo/helm/rook-operator/README.md

```diff
@@ -0,0 +1,110 @@
+rook-operator
+=============
+
+Installs [rook-operator](https://github.com/rook/rook) to create/configure/manage Rook clusters atop Kubernetes.
+
+TL;DR;
+------
+
+```console
+$ helm repo add rook http://charts.rook.io
+$ helm install rook/rook-operator
+```
+
+Introduction
+------------
+
+This chart bootstraps a [rook-operator](https://github.com/rook/rook) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Prerequisites
+-------------
+
+-	Kubernetes 1.5+ with Beta APIs & ThirdPartyResources enabled
+
+### RBAC
+
+If role-based access control (RBAC) is enabled in your cluster, you may need to give Tiller (the server-side component of Helm) additional permissions. **If RBAC is not enabled, be sure to set `rbacEnable` to `false` when installing the chart.**
+
+1.	Create a ServiceAccount for Tiller in the `kube-system` namespace
+
+```console
+$ kubectl -n kube-system create sa tiller
+```
+
+1.	Create a ClusterRoleBinding for Tiller
+
+```console
+# kubectl v1.5.x
+$ cat <<EOF | kubectl create -f -
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+EOF
+
+# kubectl v1.6.x
+$ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+```
+
+1.	Patch Tiller's Deployment to use the new ServiceAccount
+
+```console
+$ kubectl -n kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"spec": {"serviceAccountName": "tiller"}}}}'
+```
+
+Installing the Chart
+--------------------
+
+To install the chart with the release name `rook-operator`:
+
+```console
+$ helm install --name rook-operator rook/rook-operator
+```
+
+The command deploys rook-operator on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+Uninstalling the Chart
+----------------------
+
+To uninstall/delete the `rook-operator` deployment:
+
+```console
+$ helm delete rook-operator
```

Comment:
That's correct. It will only delete the resources specified in the chart. Does the operator launch anything from its deployment?

I've started a second chart for the cluster TPR so that will need to do more cleanup